src/earn/EarnActivePools.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { EarnEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import { formatValueToDisplay } from 'src/components/TokenDisplay'
+import { useDollarsToLocalAmount } from 'src/localCurrency/hooks'
+import { getLocalCurrencySymbol } from 'src/localCurrency/selectors'
+import { useSelector } from 'src/redux/hooks'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { useTokenInfo } from 'src/tokens/hooks'
+import Logger from 'src/utils/Logger'
+import networkConfig from 'src/web3/networkConfig'
+
+const TAG = 'earn/EarnActivePools'
+
+export default function EarnActivePools() {
+  const { t } = useTranslation()
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+
+  // TODO(ACT-1268): use info from getEarnPositions
+  const poolToken = useTokenInfo(networkConfig.aaveArbUsdcTokenId)
```

Comment:
using the network config here instead of getting this as props from EarnCard because this is only temporary.